pallets/ajuna-tournament/src/lib.rs

```diff
@@ -182,14 +182,15 @@ pub mod pallet {
 
 		/// The account ID of the sub-account for a given season_id/tournament_id.
 		pub fn tournament_treasury_account_id(
-			season_id: &T::SeasonId,
-			tournament_id: &TournamentId,
+			season_id: T::SeasonId,
+			tournament_id: TournamentId,
```

Comment:
If we dereference the values inside, I prefer not having borrowed values as function args.